python/tank/descriptor/io_descriptor/git.py

```diff
@@ -228,7 +228,7 @@ def _clone_then_execute_git_commands(self, target_path, commands):
                     )
 
                     # note: it seems on windows, the result is sometimes wrapped in single quotes.
-                    output = output.strip().strip(six.ensure_binary("'"))
```

Comment:
process.subprocess_check_output returns text, not binary. I've flipped that logic as it seems more sane to expect text than binary from console output. Thoughts?